stories/Atom/Icons/icons.scss

```diff
@@ -66,4 +66,19 @@
   }
 }
 
+// For doc styling only
+#story--foundation-icons--icons > div {
+  display: flex;
+  text-align: center;
+  align-items: center;
+  box-sizing: border-box;
+  flex-direction: column;
+  width: calc(25% - 2rem);
+  justify-content: center;
+  background: $color-gray-400;
+  padding: $spacing-16 $spacing-08;
+  margin: 0 $spacing-16 $spacing-32;
+  box-shadow: 2px 2px 4px $color-gray-400;
```

Comment:
@Shubham-Sharma-77 Use `$color-gray-300` for `background` and `box-shadow` and add a border of `border: 1px solid $color-gray-400`.

Reply:
@shishirphadke Ok